addons/velero/template/testgrid/k8s-docker.yaml

```diff
@@ -82,8 +86,10 @@
       version: 1.21.x
     flannel:
       version: latest
-    longhorn:
+    openebs:
       version: latest
+      isLocalPVEnabled: true
+      localPVStorageClassName: default
```

Comment:
This one won't work, I'm sorry

Velero on non-object-store requires RWX volumes which openebs does not have

Reply:
I see `"Velero Remove Object Storage"`. Good catcher. 
I added minio.